gate-web/src/main/groovy/com/netflix/spinnaker/gate/Main.groovy

```diff
@@ -17,7 +17,6 @@
 package com.netflix.spinnaker.gate
 
 import com.netflix.spinnaker.hystrix.spectator.HystrixSpectatorConfig
-import org.springframework.boot.actuate.autoconfigure.ldap.LdapHealthIndicatorAutoConfiguration
```

Comment:
this used to be excluded because in a non LDAP deploy it was causing issues. can't remember the exact context now but I'd like to be sure this is save to remove
In fiat it got renamed to the new class rather than removed:
https://github.com/spinnaker/fiat/pull/510/files#diff-e43bfd37e143f29b0c7ec7e3479a8a83R45

Reply:
Following the trail here, the issue referenced in Fiat leads to [an issue in Spring LDAP](https://github.com/spring-projects/spring-ldap/pull/474), which was fixed in `spring-ldap:2.3.3`. We're still getting `gate-ldap` -> `spring-security-ldap:5.2.1.RELEASE` -> `spring-ldap:2.3.2.RELEASE`. I'll put the autoconfig exclusion back for now.

Reply:
Resolved in 764f729f656ac305e54661186cfda2268490a021